### PR TITLE
Move progress and activity examples to a proper place

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/917
[Problem] No progress and activity examples
[Solution] Move examples to a proper place.

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/presentation-views/index.html [hunk 1]
```diff
@@ -2,28 +2,35 @@
...
-        </header>
-        <div class="ui-content">
-            <ul class="ui-list">
+		<header class="ui-header">
+			<h2 class="ui-title">
+				Presentation views
+			</h2>
+		</header>
+		<div class="ui-content">
+            <ul class="ui-listview">
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/index.html]: Check indents please.

File: examples/wearable/UIComponents/contents/presentation-views/progress-and-activity/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	<script src="../../../lib/tau/wearable/js/tau.js"></script>
+</head>
+<body>
+    <div class="ui-page" data-go-to-the-top-button="true">
+        <header class="ui-header">
+            <h2 class="ui-title">Progress & activity</h2>
+        </header>
+        <div class="ui-content">
+            <ul class="ui-listview">
+				<li>
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/progress-and-activity/index.html:23]: Ditto.

File: examples/wearable/UIComponents/contents/controls/index.html [hunk 3]
```diff
@@ -25,11 +25,6 @@ <h2 class="ui-title">
 		</header>
 		<div class="ui-content">
 			<ul class="ui-listview">
-				<li>
-					<a href="selector/index.html">
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/controls/index.html:29]: I can not find usage of this file. How about moving it to etc or removing if we don't need it.

Comment (lslachciak) [reply examples/wearable/UIComponents/contents/controls/index.html:29]: This link was not working. That is why I removed it. There is no selector folder
https://github.com/Samsung/TAU/tree/tau_1.2/examples/wearable/UIComponents/contents/controls

Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/controls/index.html:29]: That's clear now. Thanks.

File: examples/wearable/UIComponents/contents/presentation-views/index.html (hunk 1 above)
Comment (lslachciak) [reply examples/wearable/UIComponents/contents/presentation-views/index.html]: done

File: examples/wearable/UIComponents/contents/presentation-views/progress-and-activity/index.html (hunk 2 above)
Comment (lslachciak) [reply examples/wearable/UIComponents/contents/presentation-views/progress-and-activity/index.html:23]: done

**Comment by lslachciak** [comment]
Updated according to the @grzegorz-czajkowski comments

**Review by grzegorz-czajkowski — APPROVED** [review]
LGTM.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**